code/workspaces/web-app/src/auth/auth.js

```diff
@@ -90,10 +79,10 @@ function processHash(authResponse) {
 }
 
 function processResponse(authResponse) {
-  const state = processState(authResponse.state);
+  const state = undefined;
   const appRedirect = state ? state.appRedirect : undefined;
```

Comment:
Is there no authResponse.state anymore?  At the moment, I think appRedirect will always be undefined.